### ISSUE TITLE
Visual indication of mute

### ISSUE THREAD

**Issue by ytzelf** [Belphemur/SoundSwitch#540]
**Is your feature request related to a problem? Please describe.**
There's no way to know quickly if the computer's on mute when default Windows soud icon is disabled

**Describe the solution you'd like**
Would be good to have a visual indication in task bar that the sound is muted (similar to what windows default icon does)

**Describe alternatives you've considered**
Reenabling windows sound icon but its redundant w/ SoundSwitch

**Comment by Belphemur** [comment]
Hello,

Isn't Windows doing that already ?
Windows always shows the volume and state of the current default sound device in the systray. I have difficulty to see what SoundSwitch could add to that ?

**Comment by lgoose** [comment]
Especially with the new (**amazing**) feature of shortcut to mute it would be amazing if "any" icon status could have a crossed X or something like that when the input is muted.

**Comment by jhonathas** [comment]
It would be a very good indication that the microphone is muted. I currently use an app called MicMute to get feedback on when you're muted or not.

Unmuted
![image](https://user-images.githubusercontent.com/28179/145657184-e51f3eb6-7a7d-49bc-9a46-9b1a9a166b67.png)


Muted
![image](https://user-images.githubusercontent.com/28179/145657204-faa0eac5-93c9-41d5-8a41-9297e34801c6.png)

**Comment by entropie** [comment]
I would love an indication too.

Single red/green(transparent) dot in display corner that indicates mic state.

**Comment by AndreyFrigoris** [comment]
I agree, it would be just great to see the state of the microphone in the tray
